src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -218,6 +221,8 @@ def load_arguments(self, _):
         c.argument('upload_type', arg_type=get_enum_type(['Upload', 'UploadWithSecurityData']), min_api='2018-09-30',
                    help="Create the disk for upload scenario. 'Upload' is for Standard disk only upload. 'UploadWithSecurityData' is for OS Disk upload along with VM Guest State. Please note the 'UploadWithSecurityData' is not valid for data disk upload, it only to be used for OS Disk upload at present.")
         c.argument('performance_plus', arg_type=get_three_state_flag(), min_api='2022-07-02', help='Set this flag to true to get a boost on the performance target of the disk deployed. This flag can only be set on disk creation time and cannot be disabled after enabled')
+        c.argument('elastic_san_resource_id', min_api='2023-04-02', options_list=['--elastic-san-resource-id', '--elastic-san-id'],
+                   help='Required if createOption is CopyFromSanSnapshot. This is the ARM id of the source elastic san volume snapshot.')
```

Comment:
Could you tell customers how to determine if their `createOption` is `CopyFromSanSnapshot`?

Reply:
If the argument passed in is --elastic-san-resource-id, createOption is CopyFromSanSnapshot.

Reply:
> Required if createOption is CopyFromSanSnapshot.

Isn't this sentence referring to if `createOption` is `CopyFromSanSnapshot`, then the `--elastic-san-resource-id` is required?